install_config/aggregate_logging.adoc

```diff
@@ -3847,3 +3850,232 @@ cluster. Edit your non-cluster logging service (for example, `logging-es`,
+----
+
+Deployment failure can happen for a number of transitory reasons, such as
+the image pull taking too long or nodes being unresponsive. 
+
+Examine the deployer pod logs for possible reasons or attempt to redeploy:
+
+----
+$ oc deploy --latest logging-es-2e7ut0iq
+----
+
+Or, attempt to scale up the existing deployment:
+
+----
+$ oc scale --replicas=1 logging-es-2e7ut0iq-1
+----
+
+If the problem persists, examine the pod, events, and systemd unit
+logs to determine the source of the problem.
+
+*Image pull fails*
+
+If you specify an `openshift_logging_image_prefix` that results in images being defined 
+that does not exist, you will receive a corresponding error message, similar to the following,
+typically after creating the deployer.
+
+----
+$oc get pods
+
+NAME                     READY     STATUS                                                                                       RESTARTS   AGE
+logging-fluentd-1ub9k    0/1       Error: image registry.access.redhat.com:5000/openshift3logging-fluentd:latest not found      0          1m
+----
+
+In this example, for the intended image name
+`registry.access.redhat.com:5000/openshift3logging-fluentd:latest` needs a trailing `/`.
+
+Update the inventory file and follow the `openshift-ansible` instructions
+to re-run the `openshift_logging` role.
+
+*Cannot resolve kubernetes.default.svc.cluster.local*
+
+This internal alias for the master must be resolvable by the included
+DNS server on the master. Depending on your platform, you can run the `dig` command (for example, in a container) against the master to
+check whether this is the case:
+
+----
+$ dig kubernetes.default.svc.cluster.local @localhost
+[...]
+;; QUESTION SECTION:
+;kubernetes.default.svc.cluster.local. IN A
+
+;; ANSWER SECTION:
+kubernetes.default.svc.cluster.local. 30 IN A   172.30.0.1
+----
+
+Older versions of cluster logging did not automatically define this internal
+alias for the master. You might need to upgrade your cluster in order to
+use aggregated logging. If your cluster is up to date, there might be
+a problem with your pods reaching the SkyDNS resolver at the master
+or the pod could have been blocked from running. You must resolve this
+problem before deploying again.
+
+*Cannot connect to the master or services*
+
+If DNS resolution does not return at all or the address cannot be
+connected to from within a pod (such as the *fluentd* pod), this could be an
+indication of a system firewall/network problem. You need to debug this problem.
+
+=== Troubleshooting related to ElasticSearch
+
+The following troubleshooting issues apply to the ElasticSearch components of the EFK stack.
+
+*Elasticsearch deployments never succeed and rollback to previous version*
+
+This situation typically occurs itself on {product-title} with cluster logging deployed on AWS.  Describing the Elasticsearch pods typically reveal issues re-attaching the pods
```

Comment:
Possible extra space before "Describing" and another strange line break at "pods"

s/reveal/reveals

should pods be possessive?